overrides/default.nix

```diff
@@ -3847,11 +3865,13 @@ lib.composeManyExtensions [
       # manually specify its version.
       pyee = prev.pyee.overrideAttrs (
         old: {
-          postPatch = old.postPatch or "" + ''
+          postPatch = old.postPatch or "" +
+          (if lib.versionOlder old.version "10.0.0" then
```

Comment:
You can use `lib.optionalString` here for a bit more concision.

Reply:
I can probably even skip the parenthesis, but I'm always wary about nix order of operations.